src/core/commandRunner/CommandRunner.ts

```diff
@@ -113,6 +113,12 @@ export default class CommandRunner {
         getNodeAtLocation: this.graph.getNodeAtLocation,
       };
 
+      // Process targets before the `testCaseRecorder.preCommandHook()` to make sure all marks actually exist.
+      const targets = processTargets(
```

Comment:
How come this move wasn't necessary before?

Reply:
This is just a bug I found.  has nothing to do with these changes actually. If you try reference a mark that does not exist to while you recording a test you get an undefined exception.